MovingBlocks/Terasology
# Fixed Highlight module dependency error

### Contains

Fixes #3795 . Except, for the text, that doesn't appear red yet

### How to test

Head over to the modules.txt file in any module and change the installed version in it. Whenever the dependency resolution fails, the module appears read in the advancedgamesetup screen. Click on Module details button and you will find that it will remain highlighted red in the module details page as well.
